### PR TITLE
Add tab panels to TiDB Quick Start page and optimize content and structure

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added or deleted? (Required)

Most of the content is not changed.
Follow-up: Follow up on @dveeden's comments.

Main change:

- I used shortcodes for tab panels to create two panels "Mac" and "Linux" to reorganize the page structure. 

- I bumped version to v5.0.1, the latest stable version of v5.0. And in future releases, we'll bump the version in this page to the latest stable one. This will be written in our docs release checklist. FYI @morgo

- I rewrote the beginning according to Morgan's suggestions: 1) removed the warning on the top 2) introduce what TiUP is

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/5613
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch: after cherry-picked to release-4.0, we need to update the `5.0.1` to the latest version of v4.0
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -5,25 +5,20 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
 
 # TiDB 数据库快速上手指南
 
-> **警告：**
->
-> 对于生产环境，不要使用本文介绍的方式进行部署，而应使用 [TiUP 部署 TiDB 集群](/production-deployment-using-tiup.md)。
-
-本文介绍如何快速上手体验 TiDB 分布式数据库。有以下 2 种体验方式供用户选择。
+本指南介绍如何以最快速度开始使用 TiDB 数据库。相关操作主要通过 TiUP 进行。TiUP 是 TiDB 生态系统中的一个包管理工具.通过 TiUP，可以只用一行命令就运行任意 TiDB 集群组件。
```
Comment (TomShawn) [inline quick-start-with-tidb.md:8]:
```suggestion
本指南介绍如何最快上手体验 TiDB 数据库。要上手 TiDB 数据库，你将使用到 TiUP，即 TiDB 生态系统中的一个包管理工具。通过 TiUP，你只需执行一行命令就可运行任意 TiDB 集群组件。
```

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -5,25 +5,20 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
...
 
-- [第一种：使用 TiUP Playground 快速部署本地测试环境](#第一种使用-tiup-playground-快速部署本地测试环境)
-- [第二种：使用 TiUP cluster 在单机上模拟生产环境部署步骤](#第二种使用-tiup-cluster-在单机上模拟生产环境部署步骤)
+如需部署本地生产集群，可参考[在生产环境中部署 TiDB 指南](/production-deployment-using-tiup.md)。如需在 Kubernetes 上部署 TiDB，可参考[快速上手 TiDB Operator](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/get-started)。如需在云上管理 TiDB，可参考 [TiDB Cloud 快速上手指南](https://docs.pingcap.com/tidbcloud/beta/tidb-cloud-quickstart)。
```
Comment (TomShawn) [inline quick-start-with-tidb.md:10]:
```suggestion
如需部署本地生产集群，请参考[在生产环境中部署 TiDB 指南](/production-deployment-using-tiup.md)。如需在 Kubernetes 上部署 TiDB，请参考[快速上手 TiDB Operator](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/get-started)。如需在云上管理 TiDB，请参考 [TiDB Cloud 快速上手指南](https://docs.pingcap.com/tidbcloud/beta/tidb-cloud-quickstart)。
```

File: quick-start-with-tidb.md [hunk 3]
```diff
@@ -37,7 +32,108 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    source .bash_profile
+    ```
+
+3. 在当前 session 执行以下命令启动集群。
+
+    - 直接运行 tiup playground 命令会运行最新版本的 TiDB 集群，其中 TiDB、TiKV、PD 和 TiFlash 实例各 1 个：
```
Comment (TomShawn) [inline quick-start-with-tidb.md:45]:
```suggestion
    - 直接运行 `tiup playground` 命令会运行最新版本的 TiDB 集群，其中 TiDB、TiKV、PD 和 TiFlash 实例各 1 个：
```

File: quick-start-with-tidb.md [hunk 4]
```diff
@@ -37,7 +32,108 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
...
+        To connect TiDB: mysql --host 127.0.0.1 --port 4000 -u root
+        To connect TiDB: mysql --host 127.0.0.1 --port 4001 -u root
+        To view the dashboard: http://127.0.0.1:2379/dashboard
+        To view the monitor: http://127.0.0.1:9090
+        ```
+
+        > **注意：**
+        >
+        > 以这种方式执行的 playground，在结束部署测试后 TiUP 会清理掉原集群数据，重新执行该命令后会得到一个全新的集群。
+        > 若希望持久化数据，可以执行 TiUP 的 `--tag` 参数：`tiup --tag <your-tag> playground ...`，详情参考 [TiUP 参考手册](/tiup/tiup-reference.md#-t---tag-string)。
```
Comment (TomShawn) [inline quick-start-with-tidb.md:74]:
```suggestion
        > + 以这种方式执行的 playground，在结束部署测试后 TiUP 会清理掉原集群数据，重新执行该命令后会得到一个全新的集群。
        > + 若希望持久化数据，可以执行 TiUP 的 `--tag` 参数：`tiup --tag <your-tag> playground ...`，详情参考 [TiUP 参考手册](/tiup/tiup-reference.md#-t---tag-string)。
```

File: quick-start-with-tidb.md [hunk 5]
```diff
@@ -37,7 +32,108 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
...
+
+        ```shell
+        mysql --host 127.0.0.1 --port 4000 -u root
+        ```
+
+5. 通过 <http://127.0.0.1:9090> 访问 TiDB 的 Prometheus 管理界面。
+
+6. 通过 <http://127.0.0.1:2379/dashboard> 访问 [TiDB Dashboard](/dashboard/dashboard-intro.md) 页面，默认用户名为 `root`，密码为空。
+
+7. （可选）[将数据加载到 TiFlash](/tiflash/use-tiflash.md) 进行分析.
```
Comment (TomShawn) [inline quick-start-with-tidb.md:98]:
```suggestion
7. （可选）[将数据加载到 TiFlash](/tiflash/use-tiflash.md) 进行分析。
```

File: quick-start-with-tidb.md [hunk 6]
```diff
@@ -5,25 +5,121 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
...
+
+    ```shell
+    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+    ```
+
+2. 声明全局环境变量。
+
+    > **注意：**
+    >
+    > TiUP 安装完成后会提示对应的 profile 文件的绝对路径，以下 source 操作需要根据实际位置进行操作。
```
Comment (TomShawn) [inline quick-start-with-tidb.md:35]:
```suggestion
    > TiUP 安装完成后会提示对应 `profile` 文件的绝对路径。在执行以下 `source` 命令前，需要根据实际的 `profile` 文件位置修改命令。
```

File: quick-start-with-tidb.md [hunk 7]
```diff
@@ -5,25 +5,121 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    source .bash_profile
+    ```
+
+3. 在当前 session 执行以下命令启动集群。
+
+    - 直接运行 `tiup playground` 命令会运行最新版本的 TiDB 集群，其中 TiDB、TiKV、PD 和 TiFlash 实例各 1 个：
```
Comment (TomShawn) [inline quick-start-with-tidb.md:45]:
```suggestion
    - 直接执行 `tiup playground` 命令会运行最新版本的 TiDB 集群，其中 TiDB、TiKV、PD 和 TiFlash 实例各 1 个：
```

File: quick-start-with-tidb.md [hunk 8]
```diff
@@ -5,25 +5,121 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
...
+
+5. 通过 <http://127.0.0.1:9090> 访问 TiDB 的 Prometheus 管理界面。
+
+6. 通过 <http://127.0.0.1:2379/dashboard> 访问 [TiDB Dashboard](/dashboard/dashboard-intro.md) 页面，默认用户名为 `root`，密码为空。
+
+7. （可选）[将数据加载到 TiFlash](/tiflash/use-tiflash.md) 进行分析。
+
+8. 测试完成之后，可以通过执行以下步骤来清理集群：
+
+    1. 通过 <kbd>ctrl</kbd> + <kbd>c</kbd> 停掉进程
```
Comment (TomShawn) [inline quick-start-with-tidb.md:102]:
```suggestion
    1. 通过按下 <kbd>ctrl</kbd> + <kbd>c</kbd> 键停掉进程。
```

File: quick-start-with-tidb.md [hunk 9]
```diff
@@ -37,7 +133,7 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
 
     > **注意：**
     >
-    > TiUP 安装完成会提示对应的 profile 文件的绝对路径，以下 source 操作需要根据实际位置进行操作。
+    > TiUP 安装完成后会提示对应的 profile 文件的绝对路径，以下 source 操作需要根据实际位置进行操作。
```
Comment (TomShawn) [inline quick-start-with-tidb.md:136]:
```suggestion
    > TiUP 安装完成后会提示对应 `profile` 文件的绝对路径。在执行以下 `source` 命令前，需要根据实际的 `profile` 文件位置修改命令。
```

File: quick-start-with-tidb.md [hunk 10]
```diff
@@ -102,7 +198,7 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
 
 7. （可选）[将数据加载到 TiFlash](/tiflash/use-tiflash.md) 进行分析。
 
-8. 测试之后，可以通过执行以下步骤来清理群集：
+8. 测试完成之后，可以通过执行以下步骤来清理集群：
 
     1. 通过 <kbd>ctrl</kbd> + <kbd>c</kbd> 停掉进程
```
Comment (TomShawn) [inline quick-start-with-tidb.md:203]:
```suggestion
    1. 通过按下 <kbd>ctrl</kbd> + <kbd>c</kbd> 键停掉进程。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by King-Dylan** [comment]
@lilyjazz /cc

**Review by King-Dylan — APPROVED** [review]

**Review by lilyjazz — APPROVED** [review]
LGTM

**Pull request merged**